Examples/UIExplorer/js/UIExplorerExampleList.js

```diff
@@ -215,6 +216,7 @@ const styles = StyleSheet.create({
     borderRadius: 3,
     borderWidth: 1,
     paddingLeft: 8,
+    paddingVertical: 0,
```

Comment:
I think android has some padding by default (probably to offset for the underline)